gradle/libs.versions.toml

```diff
@@ -5,6 +5,11 @@ micronaut = "3.8.2"
 micronaut-test = "3.0.5"
 groovy = "3.0.8"
 spock = "2.3-groovy-3.0"
+awaitility = "4.2.0"
 
 [libraries]
+awaitility = { module = "org.awaitility:awaitility", version.ref = "awaitility" }
```

Comment:
Awaitility is incompatible with Groovy 4, and will make merging up more complex.

I suggest changing to Spock's `PollingConditions` now

Reply:
The tests are in java ☕️